metrics/examples/prometheus/install/prometheus-rules.yaml

```diff
@@ -48,7 +56,46 @@ spec:
         severity: warning
       annotations:
         summary: 'Kafka offline log directories'
-        description: 'There are {{ $value }} offline log directoris on {{ $labels.kubernetes_pod_name }}'
+        description: 'There are {{ $value }} offline log directories on {{ $labels.kubernetes_pod_name }}'
+    - alert: ScrapeProblem
+      expr: up{job="kubernetes-services",kubernetes_namespace!~"openshift-.+"} == 0
+      for: 3m
+      labels:
+        severity: major
+      annotations:
+        summary: 'Prometheus was unable to scrape metrics from {{ $labels.kubernetes_pod_name }}/{{ $labels.instance }}'
+        description: 'Prometheus has not been able to scrape metrics from {{ $labels.kubernetes_pod_name }}/{{ $labels.instance }} for more than 3 minutes'
+    - alert: ClusterOperatorContainerDown
+      expr: count((container_last_seen{container_name="strimzi-cluster-operator"} > (time() - 90))) < 1 or absent(container_last_seen{container_name="strimzi-cluster-operator"})
+      for: 1m
+      labels:
+        severity: major
+      annotations:
+        summary: 'Container strimzi-cluster-operator is down'
```

Comment:
Could we add pod name here? the _Container strimzi-cluster-operator_ is technically correct, but might not work. Or maybe some friendlier message "Cluster Operator is down".